classes/World.py

```diff
@@ -0,0 +1,15 @@
+import clustering.scripts as clustering_scripts
+
+
+class World:
+    def __init__(self, shift_duration: int, sample_size=100, number_of_clusters=200):
+        self.shift_duration = shift_duration
+        self.state = clustering_scripts.get_initial_state(
+            sample_size=sample_size, number_of_clusters=number_of_clusters
+        )
+        self.stack = []
```

Comment:
Shouldn't the stack be initialized with a GenerateScooterTrips event and the vehicle's first arrival?

Reply:
Yes, it should, but I don't have that event in this branch